R/install.R

```diff
@@ -130,11 +130,8 @@
     TRUE
 }
 
-.resolve_npkgs <- function(installed, repos, type = getOption("pkgType")) {
-    contribUrl <- contrib.url(repos, type=type)
-
-    availPkgs <- available.packages(contribUrl, type=type)
-    sum(rownames(installed) %in% availPkgs)
+.resolve_npkgs <- function(valid) {
```

Comment:
The convention is to name functions to indicate the source file they are from, so `.install_resolve_npkgs()`. Maybe a better name is `.install_n_invalid_pkgs()` or something, rather than 'resolve'.